Packs/Gurucul/Integrations/GuruculGRA/GuruculGRA.py

```diff
@@ -146,12 +147,44 @@ def fetch_incidents_open_cases(client: Client, max_results: int, last_run: Dict[
             incident_created_time = datetime.now().timestamp()
             incident_created_time_ms = incident_created_time * 1000
             record['incidentType'] = 'GRACase'
-            inc = {
-                'name': record.get('entity'),
-                'occurred': timestamp_to_datestring(incident_created_time_ms),
-                'rawJSON': json.dumps(record)
-            }
-            incidents.append(inc)
+            anomalies = record.get('anomalies')
+            demisto.info(anomalies)
+            if record.get('caseId') is not None:
```

Comment:
can you explain this condition? will this cause for cases that were fetched before not to be fetched anymore? is this a bug fix?

Reply:
Previously blank records creating when API will not return any result,
to stop this we added extra conditions so it will not create blank records